assets/javascripts/discourse/controllers/admin-plugins-explorer.js.es6

```diff
@@ -81,6 +97,15 @@ export default Ember.Controller.extend({
     this.set("loading", true);
```

Comment:
I think a lot of this can be amended to stop all this `.get` and `.set`, cause we this.selectedItem is already assumed to be there in the return. So `this.selectedItem.group_ids && this.selectedItem.group_ids.length < 1` and so on in this method make sense.

Reply:
All of this is unnecessary, and has been removed.